tests/rptest/chaos/redpanda_muservice.py

```diff
@@ -168,9 +168,12 @@ def _find_binary(self, service, name):
         if packaging not in {"dir"}:
             raise RuntimeError("Packaging type %s not supported" % packaging)
 
-        path = os.path.join(RedpandaMuService.V_DEV_MOUNT, "build", build_type,
+        path = os.path.join(RedpandaMuService.V_DEV_MOUNT, "vbuild", build_type,
```

Comment:
vbuild should be an env variable w/ default too.

Reply:
+1